core/sections/clause_10_labeling.adoc

```diff
@@ -16,8 +16,23 @@ Base Classes: MultiGraphic: Graphic
 
 ===== Class FontOutline
 
+* size: float
+* opacity: float
+* color: Color
+
 ===== Class Font
-* outline property
+
```

Comment:
Here we just want to add _outline_ only. The Font class is already described in 8-vector.

Reply:
removed unnecessary/redundant properties